### ISSUE TITLE
Unable to npm install - ERESOLVE unable to resolve dependency tree

### ISSUE THREAD

**Issue by nonoumasy** [MichalSzorad/react-social-login-buttons#53]
npm version: 7.5.4
```
pm ERR! Could not resolve dependency:
npm ERR! peer react@"^15.0.0 || ^16.0.0" from react-social-login-buttons@3.1.2
npm ERR! node_modules/react-social-login-buttons
npm ERR!   react-social-login-buttons@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
```

**Comment by MaxiDave** [comment]
This will be solved once #51 is accepted.

**Comment by MichalSzorad** [comment]
That should be now resolved in version 3.1.3. Thanks for reporting this issue!
